### DISCUSSION TITLE
How to transfer issues from unarchived repo?

### DISCUSSION THREAD

**Discussion by eitsupi** [community/community#12160] [Projects and Issues]
I cannot transfer issues from one unarchived repository to another repo.
I'm wondering if there is something to do with between unarchiving and not being able to move an issue.

**Comment by michael-k** [comment]
Same/similar here:

We cannot transfer issues from any repository to another (both in the same organization and none of them was ever archived).  We can select the target repository and click the button, but nothing happens.

**Comment by eitsupi** [comment]
After three days of waiting, I was able to move the issues.
I am wondering if the time lapse after unarchiving has resolved this issue.
